feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -126,7 +160,9 @@ fun RepairView() {
                     )
                     .fillMaxWidth(),
                 mimeTypes = arrayOf("video/*", "image/*"),
-                onFileSelected = {/* TODO - File Button - Add functionality to save files */ }
+                onFileSelected = {
+                    repairViewModel.getImageOrVideoFileFromView(it)
```

Comment:
we need to use hight order function here and send the event to the top

Reply:
Modified! Thank you for your help